accounts/instrument_details.go

```diff
@@ -0,0 +1,68 @@
+package accounts
+
+import (
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/instruments"
+)
+
+type InstrumentDetail string
+
+type (
+	InstrumentDetails interface {
+		GetType() string
+	}
+
+	InstrumentDetailsFasterPayments struct {
+		AccountNumber string `json:"account_number,omitempty"`
+		BankCode      string `json:"bank_code,omitempty"`
+	}
+
+	InstrumentDetailsSepa struct {
+		Iban     string `json:"iban,omitempty"`
+		SwiftBic string `json:"swift_bic,omitempty"`
+	}
+)
+
+type (
+	PaymentInstrumentDetailsResponse struct {
+		HttpMetadata       common.HttpMetadata        `json:"http_metadata,omitempty"`
```

Comment:
We usually don't add the `json` and `omitempty` tags in HttpMetadata

Reply:
👍